src/site/_collections/hooks/utils.js

```diff
@@ -56,6 +56,8 @@ const index = (items, href, testItems) => {
     itemsWithPosts = items.filter((item) => item.elements.length > 0);
   }
 
+  itemsWithPosts.sort((a, b) => a.title.localeCompare(b.title));
```

Comment:
Is this alphabetizing the author's posts?

Reply:
if so, we may want to leave them in publish order

Reply:
This doesn't sort the posts, rather the tags and the authors themselves.